pkg/pagerduty/service.go

```diff
@@ -453,6 +453,26 @@ func parseIncidentNumbers(incidents []pdApi.Incident) []uint {
 	return incidentNumbers
 }
 
+// generateServiceName checks if FedRamp is enabled. If it is, it returns
+// an anonymized PD service name.
+func generatePDServiceName(data *Data) string {
+	if config.IsFedramp() {
+		return data.ClusterID
+	} else {
+		return data.ServicePrefix + "-" + data.ClusterID + "." + data.BaseDomain + "-hive-cluster"
+	}
+}
+
+// generateServiceDescription checks if FedRamp is enabled. If it is, it returns
+// an empty PD service description
+func generatePDServiceDescription(data *Data) string {
+	if config.IsFedramp() {
+		return ""
```

Comment:
Why leave this blank?

Reply:
As mentioned in my post above, I removed this strictly to minimize the amount of information shared to an external service for a FedRAMP cluster.